pb-crm-api/src/app.module.ts

```diff
@@ -20,6 +21,7 @@ import {ConfigModule} from "@nestjs/config";
                 "entities": ["dist/db/entities/**/*.entity{.js, .ts}"],
                 "synchronize": true,
                 "logging": true,
+                "dropSchema": true
```

Comment:
In local mode, the database is refreshed (cleared) **every single time** the api starts or reloads